api/odilo.py

```diff
@@ -65,12 +368,45 @@ class OdiloAPI(BaseOdiloAPI, BaseCirculationAPI, HasSelfTests):
     }
 
     def __init__(self, _db, collection):
-        super(OdiloAPI, self).__init__(_db, collection)
+        # super(OdiloAPI, self).__init__(_db, collection)
```

Comment:
why is this commented out?

Reply:
I meant to delete it--just fixed that!  (Because I merged the two separate Odilo classes, this no longer has a parent class (other than Object) to be calling super on.)